pages/design/ui-patterns/button-usage.mdx

```diff
@@ -0,0 +1,117 @@
+    </BorderBox>
+    <Text fontWeight='bold' color='green.5'>Do:</Text> Use sentence case 
+  </Box>
+  <Box width='100%' mr='2'>
+    <BorderBox border='1px solid' borderColor='red.5' mb="2">
+      <img src="https://user-images.githubusercontent.com/24916540/52501272-d3330080-2b94-11e9-8829-7d12aae36ba1.png"/>
+    </BorderBox>
+    <Text fontWeight='bold' color='red.5'>Don't:</Text> Use all-caps or other text formats
+  </Box> 
+</Flex>
+
+### Keep Button Text Concise
+When designing buttons, keep the text as short but descriptive as possible. Buttons are to be understood by their type and text; if further clarity is required, consider adding an octicon or adding a text description. Never wrap text within buttons. 
+
+### Adaptable Full-Width 
+In certain scenarios, it may be appropriate for a button’s width to be extended to fill their parent container. Use this style sparingly, but some general, appropriate use-cases include:
+- Modals with a singular call-to-action
+- Responsive behavior of buttons when scaled down
+
+<Flex>
+  <Box width='100%' mr='2'>
+    <BorderBox border='1px solid' borderColor='green.5' mb="2">
+      <img src="https://user-images.githubusercontent.com/24916540/52501273-d3330080-2b94-11e9-906c-27d9f745f0c5.png"/>
+    </BorderBox>
+    <Text fontWeight='bold' color='green.5'>Do:</Text> Keep text brief, specific, and descriptive
+  </Box>
+  <Box width='100%' mr='2'>
+    <BorderBox border='1px solid' borderColor='red.5' mb="2">
+      <img src="https://user-images.githubusercontent.com/24916540/52501274-d3330080-2b94-11e9-823d-61964966cc7b.png"/>
+    </BorderBox>
+    <Text fontWeight='bold' color='red.5'>Don't:</Text> Use lengthy, wordy text and/or wrap it within the button
+  </Box> 
+</Flex>
+
+### Text Descriptions
+In certain scenarios, buttons that perform very significant actions require accompanying text around them to let the user fully understand its intent. Use text labels sparingly, as buttons should generally be able to be understood on their own.
+
+<Flex>
+  <Box width='50%' mr='2'>
+    <BorderBox border='1px solid' borderColor='green.5' mb="2">
+      <img src="https://user-images.githubusercontent.com/24916540/52501400-1b522300-2b95-11e9-8c47-71af68cfaeec.png"/>
+    </BorderBox>
+    <Text fontWeight='bold' color='green.5'>Do:</Text> Keep text brief, specific, and descriptive
+  </Box> 
+</Flex>
+
+### Use Octicons to Clarify Actions
+Use octicons in conjunction with buttons to clarify an action if text alone is not enough to explain its functionality. Octicons can also be used to signal an interaction, for example, the `triangle-down` octicon to open a dropdown.
+
+Octicons can be used to call attention to the button’s action and make it stand out through consistent usage. For example, GitHub products use the repo icon in a button and then repeat its use throughout the "new repo" flow - this repetition and continuity is helps people learn our visual language. 
```

Comment:
I feel like this could be reworded. This isn't quite right, but maybe something like:

> In some cases, Octicons are used to create an identifiable visual indicator for a common action or paradigm. For example, the repo icon is used next to repo links, the New Repo button, and repeated in the New Repo flow. This repetition and continuity helps people learn and identify our visual language.